src/MonitorProfiler/Utilities/TypeNameUtilities.cpp

```diff
@@ -183,6 +183,9 @@ HRESULT TypeNameUtilities::GetTypeDefName(NameCache& nameCache, ModuleID moduleI
         }
 
         WCHAR wName[256];
+        WCHAR wNamespace[256];
```

Comment:
This isn't necessary now and should be removed. `tstring` should have a parameterless ctor to produce an initial empty string.